android/app/src/main/java/com/votlin/android/ui/activity/TalksActivity.kt

```diff
@@ -44,7 +46,10 @@ class TalksActivity : RootActivity<TalksView>(), TalksView {
         viewPagerAdapter.addFragment(getString(R.string.business), TalksListFragment.newInstance(Track.BUSINESS))
         viewPagerAdapter.addFragment(getString(R.string.development), TalksListFragment.newInstance(Track.DEVELOPMENT))
         viewPagerAdapter.addFragment(getString(R.string.maker), TalksListFragment.newInstance(Track.MAKER))
-        viewPager.adapter = viewPagerAdapter
+        viewPager.apply {
+            adapter = viewPagerAdapter
+            offscreenPageLimit = 4
```

Comment:
Please, avoid hardcoded numbers!

Reply:
I could extract it to a constant, but I don't know if that change give some value because this number is only used here and is dependent from tabs number. I mean, this tabs are hardcoded some lines above: 

`viewPagerAdapter.addFragment(getString(R.string.all), TalksListFragment.newInstance(Track.ALL))
        viewPagerAdapter.addFragment(getString(R.string.business), TalksListFragment.newInstance(Track.BUSINESS))
        viewPagerAdapter.addFragment(getString(R.string.development), TalksListFragment.newInstance(Track.DEVELOPMENT))
        viewPagerAdapter.addFragment(getString(R.string.maker), TalksListFragment.newInstance(Track.MAKER))`

So, maybe the right solution is to have an array with tab fragments, and use the length of it for this **offscreenPageLimit**. What do you think about that?